### PR TITLE
Create React components that rendering arrangement JSON objects

### PR DESCRIPTION
**Pull request by Jeffchiucp**
- Create React components that rendering arrangement JSON objects
- Display the arrangement object as the following:

screen shot 2018-10-04 at 4 37 59 pm

TODO:
- Add the Backend URL to the FE and test the connection

### CODE REVIEW

**Comment by Jeffchiucp** [comment]
@deonjo 
Here's some question for us to think about:
How do you want us to render the snapshots?

Can you render the containers and items with your JSON data?

and do we need multiple snapshots?

**Comment by Jeffchiucp** [comment]
update the and testing the connection with BE and FE 
screen shot 2018-10-06 at 10 19 03 am

**Comment by Jeffchiucp** [comment]
i tested the react app and it renders the following JSON object correctly:
https://arrangement-server.herokuapp.com/arrangement/zzzzMEDEA

**Comment by Jeffchiucp** [comment]
the new commit makes the changes to the React component, and it looks like this:
screen shot 2018-10-10 at 9 40 03 pm
you can see the arrangement JSON  snapshot here: https://arrangement-server.herokuapp.com/api/v1/arrangement/zzzzEPQPQ

**Comment by Jeffchiucp** [comment]
@deonjo requested for code review and merge the Pull request

File: src/components/containercollection/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+export class ContainerCollection extends Component {
+  render () {
+    return (
+      <div className="containercollection">
+        <Grid container spacing={24}>
+          {
+            this.props.containers.map((container) => {
+              return (
+                <Grid item xs={12} sm={4} md={3} key={container._id}>
+                  <Container container={container} snapshot={this.props.snapshot} items={this.props.items} />
```
Comment (deonjo) [inline src/components/containercollection/index.js]: what's this?

File: src/components/containercollection/index.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            })
+          }
+        </Grid>
+      </div>
+    )
+  }
+}
+
+ContainerCollection.propTypes = {
+  snapshot: PropTypes.shape({
```
Comment (deonjo) [inline src/components/containercollection/index.js]: why is the snapshot within the container?

**Pull request merged**